CHANGES.md

```diff
@@ -7,6 +7,7 @@ Next release
 
 Features
 --------
+* [#569](https://github.com/java-native-access/jna/pull/569): Added `com.sun.jna.platform.win32.Winspool.PRINTER_INFO_2` support and GetPrinter function in `com.sun.jna.platform.win32.Winspool` [@IvanRF](https://github.com/IvanRF)
```

Comment:
The syntax of your line and the one below is different, it's `- [@](...).`, if you don't mind fixing please.